File: latihanB/python/Product.py
class Product:
    def __init__(self):
        self.price = ""
        self.idProduct =""
    
    def setPrice(self, price):
        self.price = price
    
    def getPrice(self):
        return self.price

    def setIdProduct(self, idProduct):
        self.idProduct = idProduct
    
    def getIdProduct(self):
        return self.idProduct

File: latihanB/java/Product.java
class Product {
    private String price;
    private String idProduct;

    Product() {
        this.price = "";
        this.idProduct = "";
    }

    void setPrice(String price) {
        this.price = price;
    }

    String getPrice() {
        return this.price;
    }

    void setIdProduct(String idProduct) {
        this.idProduct = idProduct;
    }

    String getIdProduct() {
        return this.idProduct;
    }
}